.github/workflows/pypi-build-artifacts.yml

```diff
@@ -75,13 +75,10 @@ jobs:
           CIBW_ARCHS: "auto64"
           CIBW_PROJECT_REQUIRES_PYTHON: ">=3.9,<3.13"
           CIBW_TEST_REQUIRES: "pytest==7.4.2 moto==5.0.1"
-          CIBW_TEST_EXTRAS: "s3fs,glue"
```

Comment:
Nice, wasn't aware that we could drop those. Should speed it up also quite a bit 👍